extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -228,8 +233,11 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 			ref: getRefParam()
 		};
 
-		if (pvs) {
-			params.pv = pvs.toString();
+		if (skin && skin !== 'mercury') {
+			params.pv = pvs;
+		} else {
+			// this is a temporary solution to test if Mercury handles pvs well without affecting actual ads
```

Comment:
Worry not, this will not be forgotten (since it's crucial for the newly invented mobile interstitial ads). We'll enable this when working on other Mercury-related tickets. The task itself is not worth having a JIRA ticket for it (with all the overhead that comes with it).